.circleci/config.yml

```diff
@@ -75,14 +75,27 @@ commands:
           destination: test_results
 
 jobs:
-  unit_test:
+  linting_checks:
     <<: *job_defaults
     steps:
       - test_setup
```

Comment:
Refactoring this would cut the build time down substantially. Linting does not need all the steps in `test_setup` (e.g. google cloud sdk is not needed).